pkg/controller/operators/catalog/operator.go

```diff
@@ -2173,6 +2175,15 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 			}
 			return nil
 		}(i, step); err != nil {
+			s := step
```

Comment:
is this doing what we expect?

Reply:
I don't think aliasing this is necessary.